metadata/metadata/routes.py

```diff
@@ -41,7 +41,7 @@ def create_submission() -> Union[str, Response]:
     )
 
 
-@blueprint.route('/<string:submission_id>', methods=['GET'])
+@blueprint.route('/<string:submission_id>/', methods=['GET'])
```

Comment:
The openapi.yaml says that there should be trailing slashes, so I added them. But I'm not sure how I feel about them.

Reply:
I'm not crazy about trailing slashes either. A truly RESTful API would allow you to manually content negotiate using that `/ID.json` `/ID.html` etc. with the trailing slash, negotiation is forced onto the request headers, I think.